card/lib/card/auth.rb

```diff
@@ -214,7 +213,9 @@ def always_ok_usr_id? usr_id
       def createable_types
         type_names = Auth.as_bot do
           Card.search type: Card::CardtypeID, return: :name,
-                      not: { codename: ['in'] + NON_CREATEABLE_TYPES }
+                      not: {
+                        codename: ['in'] + Card.config.non_createable_types
+                      }
```

Comment:
made NON_CREATEABLE_TYPES a config option so that I can add something in a wikirate mod. The default is set in cardio.rb